geode-core/src/test/java/org/apache/geode/distributed/internal/InternalLocatorTest.java

```diff
@@ -47,6 +49,16 @@
   private AgentUtil agentUtil = mock(AgentUtil.class);
   private HttpService httpService = mock(HttpService.class);
 
+  @After
+  public void cleanup() {
+    if (internalLocator != null) {
+      try {
```

Comment:
You should be able to delete the try-catch here.

Reply:
done